spec/automation/unit/method_validation/quota_source_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "spec_helper"
+include QuotaHelper
+
+def run_automate_method(provision_request)
+  attrs = []
+  attrs << "MiqProvisionRequest::miq_provision_request=#{@miq_provision_request.id}&" \
+         "MiqRequest::miq_request=#{@miq_provision_request.id}" if provision_request
+  ws = MiqAeEngine.instantiate("/ManageIQ/system/request/Call_Instance?namespace=System/CommonMethods&" \
```

Comment:
@tinaafitz 
MiqAeEngine.instantiate needs a user object to be passed in.
Also we don't need the ws variable, the MiqAeEngine.instantiate returns a workspace which can be returned directly to the caller